pkg/kstatus/polling/aggregator/aggregator.go

```diff
@@ -9,22 +9,34 @@ import (
 	"sigs.k8s.io/cli-utils/pkg/kstatus/wait"
 )
 
-// BasicAggregator implements StatusAggregator.
-// Aggregate status will be Current when all polled
-// resources are either Current or NotFound.
-// TODO: Treating resources that doesn't exist as Current is
-// weird. But it kinda does make sense when we track
-// resources that are deleted/pruned. We should see if
-// there is a better way to handle this.
-type BasicAggregator struct {
+// NewAllCurrentAggregator returns a new aggregator that will consider
+// the set to be completed when all resources have reached the Current status.
+func NewAllCurrentAggregator(identifiers []wait.ResourceIdentifier) *genericAggregator {
+	return newGenericAggregator(identifiers, status.CurrentStatus)
+}
+
+// NewAllNotFoundAggregator returns a new aggregator that will consider
+// the set to be completed when all resources have reached the Current status.
```

Comment:
How is this different than the NewAllCurrentAggregator?

Reply:
That was a cut-and-paste error. I have updated the comment.